core/translate/expr.rs

```diff
@@ -1848,15 +1848,24 @@ pub fn translate_expr(
 
                     MathFuncArity::Binary => {
                         let args = expect_arguments_exact!(args, 2, math_func);
-                        let reg1 = program.alloc_register();
-                        let _ =
-                            translate_expr(program, referenced_tables, &args[0], reg1, resolver)?;
-                        let reg2 = program.alloc_register();
-                        let _ =
-                            translate_expr(program, referenced_tables, &args[1], reg2, resolver)?;
+                        let start_reg = program.alloc_registers(2);
```

Comment:
This is like the 100th bug involving this specific thing

Reply:
Yes :(

I tried to fix this more globally with PR https://github.com/tursodatabase/limbo/pull/957 - but this was not enough.

But I think that having more coverage with fuzz test is good - so I think it's ok to fix them that way. But reviewing `expr.rs` code for such bugs independently also good thing.